releasenotes/notes/refactor-trace-obfuscator-config-3da22d15bd9ede0a.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Refactor & simplify APM trace obfuscator config
```

Comment:
Is this change backward compatible ? How will it impact users ? We need to explain more what this means for users,

Reply:
No functional change, just a refactor. Removing the changelog here https://github.com/DataDog/datadog-agent/pull/5757